cmd/route-advertisement-daemon/main.go

```diff
@@ -0,0 +1,335 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"net/netip"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/jcelliott/lumber"
```

Comment:
Any reason we choose this log lib? It doesn't seem to have many stars

Reply:
There are a little few log libs support rotation, this lib was imported by most users. However, it doesn't seem active and thread-safe.

Any suggestions? Or just use the built-in log without rotation beacase i have removed periodical logs that would cause the size of the log file to grow rapidly.

Reply:
For a process in a container, it's better we log to console